source/runbooks/rotating-secrets.html.md.erb

```diff
@@ -13,13 +13,19 @@ We don't have many secrets on stored on the Modernisation Platform, but they are
-| Modernisation Platform Account Root User Password | Only used during initial platform set up, log in prevented via SCP and no password or keys set | Not stored | Disable or move account to a non SCP protected OU and follow the password reset steps |
+| PagerDuty Token | pagerduty_token | Used by PagerDuty Terraform to manage PagerDuty resources|AWS Secrets Manager | Contact Operations Engineering to issue a new token and update the secret. |
+| PagerDuty Modernisation Platform Team user | pagerduty_integration_keys | Used for dead-end notifications as all schedules need a user | Not stored | Use password reset process if needed |
+| Slack Webhook URL | slack_webhook_url | Used to post alarms to Slack | AWS Secrets Manager | Contact Operations Engineeering to issue a new incoming webhook for the `Modernisation Platform Alerts` custom Slack application.  Revoke the old incoming webhook and update the secret. |
+| GitHub MP CI User PAT | github_ci_user_pat | Used to create PRs etc in GitHub actions and deploy GitHub resources via Terraform | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| GitHub MP CI User Environments Repo PAT | github_ci_user_environments_repo_pat | Used in reusable pipelines of the modernisation-platform-environments repository. This is so that the CI user can post comments in PRs, e.g. tf plan/apply output. | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret.|
+| GitHub MP CI User Password | github_ci_user_password | Used to log in and set the PAT | AWS Secrets Manager | Log in to GitHub as the user and reset the password, update the secret |
+| Environment Management | environment_management | IDs for AWS-specific resources for environment management, such as organizational unit IDs | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| Nuke ID List | nuke_account_ids | Account IDs to be auto-nuked on weekly basis. This secret is used by GitHub actions job nuke.yml inside the environments repo, to find the Account IDs to be nuked. |	Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+| Nuke Block List | nuke_account_blocklist | Account IDs to be excluded from auto-nuke. AWS-Nuke (https://github.com/rebuy-de/aws-nuke) requires at least one Account ID to be present in this blocklist, while it is recommended to add every production account to this blocklist. | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
+|	PagerDuty user api | pagerduty_userapi_token | PagerDuty api user token | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. |
```

Comment:
What is the difference between this token and the token on line 18?  Do we need 2 tokens or can one of these be removed from secrets?

Reply:
I do not know. Both are used but this one slightly less than the other. In just 4 places and in two of those it is in uppercase. It can probably be replaced by the one above.e.g

data "aws_secretsmanager_secret" "pagerduty_user_token" {

name = "pagerduty_userapi_token"
 }

Reply:
Additional note added to explain why this is needed